test/models/deploy_group_test.rb

```diff
@@ -1,18 +1,30 @@
 require_relative '../test_helper'
 
 describe DeployGroup do
-  let(:prod_env) { environments(:production) }
+  let(:stage) { stages(:test_staging) }
+  let(:environment) { environments(:production) }
+  let(:deploy_group) { deploy_groups(:pod1) }
+
+  def self.it_expires_stage(method)
+    it "expires stages when #{method}" do
+      stage.deploy_groups << deploy_group
+      stage.update_column(:updated_at, 1.minute.ago)
```

Comment:
My suggestion was to make an old fixture - not sure what we can do here but in factorygirl you could go `let(:stage) { create(:stage, updated_at: 1.minute.ago) }` or similar